pkgs/by-name/ir/ironbar/package.nix

```
@@ -56,16 +64,31 @@ rustPlatform.buildRustPackage rec {
     gtk3
   ];
 
-  preFixup = ''
-    gappsWrapperArgs+=(
+  runtimeDeps = [
+    luajitPackages.lgi
+  ];
+
+  buildNoDefaultFeatures = features != [];
+  buildFeatures = features;
+
+  gappsWrapperArgs = ''
       # Thumbnailers
-      --prefix XDG_DATA_DIRS : "${gdk-pixbuf}/share"
-      --prefix XDG_DATA_DIRS : "${librsvg}/share"
-      --prefix XDG_DATA_DIRS : "${webp-pixbuf-loader}/share"
-      --prefix XDG_DATA_DIRS : "${shared-mime-info}/share"
+          --prefix XDG_DATA_DIRS : "${gdk-pixbuf}/share"
+          --prefix XDG_DATA_DIRS : "${librsvg}/share"
+          --prefix XDG_DATA_DIRS : "${webp-pixbuf-loader}/share"
+          --prefix XDG_DATA_DIRS : "${shared-mime-info}/share"
+
+          # gtk-launch
+          --suffix PATH : "${lib.makeBinPath [ gtk3 ]}"
+  ''
+  + (if hasFeature "cairo" then ''
+      --prefix LUA_PATH : "./?.lua;${luajitPackages.lgi}/share/lua/5.1/?.lua;${luajitPackages.lgi}/share/lua/5.1/?/init.lua;${luajit}/share/lua/5.1/\?.lua;${luajit}/share/lua/5.1/?/init.lua"
+      --prefix LUA_CPATH : "./?.so;${luajitPackages.lgi}/lib/lua/5.1/?.so;${luajit}/lib/lua/5.1/?.so;${luajit}/lib/lua/5.1/loadall.so"
```

```suggestion
    --prefix LUA_PATH : "./?.lua;${luajitPackages.lgi}/share/lua/5.1/?.lua;${luajitPackages.lgi}/share/lua/5.1/?/init.lua;${luajit}/share/lua/5.1/\?.lua;${luajit}/share/lua/5.1/?/init.lua"
    --prefix LUA_CPATH : "./?.so;${luajitPackages.lgi}/lib/lua/5.1/?.so;${luajit}/lib/lua/5.1/?.so;${luajit}/lib/lua/5.1/loadall.so"
```
Nix uses 2 spaces for indentation